advanced/hpc_compute_to_data/jupyter_notebook/Dockerfile.2

```diff
@@ -0,0 +1,28 @@
+from dwmoore/intermediate-notebook
```

Comment:
We will need to change this base image name, I think.

More specifically, we should probably describe what the other base image is (in a reusable fashion). Since this is for training, we can instruct trainees to tag the image in such a way that it's not tied to a namespace. If we would like to keep a namespace, we should use the irods namespace.

Reply:
Yes, I figured we'd need to move some things toward genericity, reusability, & toward being more official and correct. Will try to think of a name.  Also, who has docker push privilege  to the irods namespace?